src/main/java/openfasttrack/cli/CommandLineInterpreter.java

```diff
@@ -111,7 +117,8 @@ public void parse()
 
     private void handleNamedArgument(final ListIterator<String> iterator, final String argument)
     {
-        final String argumentName = argument.substring(SINGLE_CHAR_ARG_PREFIX.length());
```

Comment:
I just realized that we don't have code to distinguish between single and multiple character parameters.

As per GNU convention the following is equivalent:

```
myprogram --verbose --recursive
myprogram -v -r
myprogram -vr
```

Wrote ticket #25.